frontend/packages/dev-console/src/components/edit-application/edit-application-utils.ts

```diff
@@ -50,6 +50,25 @@ export enum CreateApplicationFlow {
   JarUpload = 'Upload JAR file',
 }
 
+export const getEditFlowPageTitle = (editflowtype: CreateApplicationFlow): string => {
+  switch (editflowtype) {
+    case CreateApplicationFlow.Git:
+      // t('devconsole~Import from Git')
+      return 'devconsole~Import from Git';
+    case CreateApplicationFlow.Dockerfile:
+      // t('devconsole~Import from Dockerfile')
+      return 'devconsole~Import from Dockerfile';
+    case CreateApplicationFlow.Container:
+      // t('devconsole~Deploy Image')
+      return 'devconsole~Deploy Image';
+    case CreateApplicationFlow.JarUpload:
+      // t('devconsole~Upload JAR file')
+      return 'devconsole~Upload JAR file';
+    default:
+      return '';
```

Comment:
We should return the flowType here as fallback. If we add an enum value and don't add the translation we should at least show the not translated version.

Reply:
For all values it's also possible to return "devconsole~" + flowType.
You can return this always this way or what's fine, just return the default value this way. Both is fine for me. If you keep the switch I see the benefit that this code doesn't expect anymore that the enum value is also the title. 👍